src/main/scala/com/typesafe/sbt/packager/rpm/RpmMetadata.scala

```diff
@@ -222,6 +223,9 @@ case class RpmSpec(meta: RpmMetadata,
     sb append ("Version: %s\n" format meta.version)
     sb append ("Release: %s\n" format meta.release)
     sb append ("Summary: %s\n" format meta.summary)
+
+    if (meta.epoch > 0) sb append ("Epoch: %d\n" format meta.epoch)
```

Comment:
I think it's better to use `Option` in such cases

Reply:
I went back and forth on that. Doing this way also protests against negative epochs, but I suppose I could do an exists or filter for the negative check. Thoughts?

Reply:
That sounds good. Option + check.

You can also add a [validation check](https://github.com/sbt/sbt-native-packager/blob/master/src/main/scala/com/typesafe/sbt/packager/rpm/RpmPlugin.scala#L105) for this a log an error or warning.

Would `rpmLint` catch this?

Reply:
I don't think rpmLint has a check for this